### PR TITLE
Viz EC2 Services to Lambda - Part 2 - Rapid Onset Flooding Probability

### PR DESCRIPTION
**Pull request by TylerSchrag-NOAA**
This PR migrates the ensemble-based Rapid Onset Flooding Probability products from the Viz EC2 to the Viz Max Values lambda function (now called Python Preprocessing). It broadly encapsulates the following changes:

**Changes**
- New rapid_onset_flooding product script in the python_preprocessing lambda function that supports both 12 hour SRF and 5 day GFS MRF configurations.
- Removal of rapid_onset_flooding pipeline and product files in the source-aws_loosa library

**Deployment Considerations:**
- Not sure if we should include this in the 2.1.4 release or not? I'm good to test/fix quickly and thoroughly next week if you want to include it. Otherwise, fine to go in the next one (wait to merge for now)
- We will need to include a ingest schema db dump when deploying to UAT.

### CODE REVIEW

File: Core/LAMBDA/viz_functions/viz_db_postprocess_sql/summaries/mrf_gfs_5day_max_high_water_probability/hucs.sql
[the remark this answers is not in the document]
Comment (CoreyKrewson-NOAA) [reply Core/LAMBDA/viz_functions/viz_db_postprocess_sql/summaries/mrf_gfs_5day_max_high_water_probability/hucs.sql:1]: as in the product is creating a "mrf_gfs_5day_high_water_probability_hucs" table but the mapx is looking for a "mrf_gfs_5day_max_high_water_probability_hucs"

**Comment by CoreyKrewson-NOAA** [comment]
@TylerSchrag-NOAA I also updated the pipeline_config file which is used by the EC2 to create the windows services

**Comment by TylerSchrag-NOAA** [comment]
> @TylerSchrag-NOAA I also updated the pipeline_config file which is used by the EC2 to create the windows services

Thank you! I totally missed that.

**Review by shawncrawley — DISMISSED** [review]
This looks good to me!

**Comment by TylerSchrag-NOAA** [comment]
> This looks good to me!

Thanks Shawn - Sorry to make you approve again, but I just realized I forgot to add the new dask lambda layer.

**Review by shawncrawley — DISMISSED** [review]
Ah yes, the dask layer. Now it looks better.

File: Core/LAMBDA/viz_functions/viz_db_postprocess_sql/summaries/mrf_gfs_5day_max_high_water_probability/hucs.sql
[the remark this answers is not in the document]
Comment (TylerSchrag-NOAA) [reply Core/LAMBDA/viz_functions/viz_db_postprocess_sql/summaries/mrf_gfs_5day_max_high_water_probability/hucs.sql:1]: You're right, I missed/botched this one somehow. Thank you!

**Review by shawncrawley — APPROVED** [review]
This looks good to me.

**Pull request merged**